### DISCUSSION TITLE
Help with Microsoft auth

### DISCUSSION THREAD

**Discussion by arrowcircle** [outline/outline#2780] [General]
Hi! I was trying to set up self hosted version of outline with microsoft login, created all credentials according to guide https://wiki.generaloutline.com/share/dfa77e56-d4d2-4b51-8ff8-84ea6608faa4
But when I click on the login button, I see microsoft account i need to choose. When I click on it, it goes somewhere and returns to the same page with this account.

Outline logs are empty, there are no errors in the browser console. What I doing wrong?
Is there are any simple way to have auth for outline?

**Comment by Limezy** [comment]
> Is there are any simple way to have auth for outline?

Building a slack app at [api.slack.com/apps ](https://api.slack.com/apps) seems to be the easiest way for Outline auth at the moment.
Are you sure you did setup correctly the environment variables in `.env` file ? Also, as far as I could check, Outline doesn't work without correctly configured https for auth.

**Reply by arrowcircle** [reply]
I redone everything 2 times even with recreating new catalog. All values are in env variables. And HTTPS is set up correctly.

**Reply by Limezy** [reply]
I guess you should put DEBUG=* in your .env and try to dig into what's happening 😞

**Reply by tommoor** [reply]
Can you post your logs?

**Reply by arrowcircle** [reply]
It does not hit outline, redirect does not happen on microsoft side. So there are no logs about that.
